utility.py

```diff
@@ -0,0 +1,135 @@
+import json
+import numpy as np
+import os, csv
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+class Utility:
+    def __init__(self, input_histogram):
+        self.hist = input_histogram
+    def __del__(self):
+        print("deling", self)
+    # =====================================
+    # calculate the probability distribution given a histogram
```

Comment:
Could you mention that this is also creating and saving a histogram?

Reply:
edited the description.